### DISCUSSION TITLE
[Issue]: nested mixed schemas with default values

### DISCUSSION THREAD

**Discussion by questionteller** [formvuelate/formvuelate#270] [Q&A] [answered]
Hello,

first of all I'd like to thank you for this awesome library as there are not many out there focussing on schema based form generation for Vue 3 so I am really glad I have stumbled upon this one!

What I've also stumbled upon is a weird issue(?) with the following schema:

[CodeSandbox of the issue](https://codesandbox.io/s/fvl-useschemaform-3-x-forked-m8lt3?file=/src/App.vue)

Once using a nested schema (with default values!) in a nested array schema, the following schema parts are getting nested as well in the form data. Could someone please elaborate on this one? Once removing the default values everything seems to work.

**Comment by marina-mosti** [comment, marked as answer]
Hey @questionteller this seems like a bug, can you reopen this as an issue?

**Reply by questionteller** [reply]
Thank you for your quick reply, see #271
